src/components/editable-input/editable-input.jsx

```diff
@@ -69,13 +69,14 @@ class InputEditable extends React.Component {
   render() {
     const { classes } = this.props;
 
-    const renderInput = this.state.editing ? (
+    const input = this.state.editing ? (
       <form className={classes.form} onSubmit={this.onSubmit}>
         <Input
           autoFocus
           className={classes.input}
           style={this.props.style}
-          variant={this.props.variant === 'primary' ? 'secondary' : 'primary'}
```

Comment:
Is the default set elsewhere? Otherwise this is a breaking change and we need to verify every place where this is used.

Reply:
It is set in the defaultProps on line 164 now instead. :)
```
EditableInput.defaultProps = {
  variant: 'secondary',
```

Reply:
Essentially, the way it was written until I broke out this into a completely new component was a workaround for the fact that `variant` was always set to `"primary"` as default for the regular input type. Now that it is it's own component, I can set the `defaultProps` the way I want them right away. 👍